modules/olm-policy-understanding.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-creating-policy.adoc
+
+[id="olm-policy-understanding_{context}"]
+= Understanding Operator installation policy
+
+Using OLM, cluster administrators can choose to specify a service account for an
+OperatorGroup so that all Operators associated with the OperatorGroup are
+deployed and run against the privileges granted to the service account.
+
+`APIService` and `CustomResourceDefinition` resources will always be created by
```

Comment:
s/will always be created/are always created
or non-passive? 
_The OLM with the `cluster-admin` role creates the `APIService` and `CustomResourceDefinition` resources._